Readme.md

```diff
@@ -1,21 +1,14 @@
-# CaPTk:  Cancer Imaging Phenomics Toolkit 
+# Binaries
 
```

Comment:
You need to completely change this file, BTW. It needs to be the same as https://github.com/CBICA/CaPTk/blob/master/Readme.md

Reply:
Got it
Updated